### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Joshezzy**
Work is still in progress, but the plan is stated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,30 @@
...
+1. Introduction
+2. Features of the project
+3. Procedure
+4. Contributors
+5. Access
+
+### Introduction
+As the population of the environment increases, the road users increases as well, thus, the need to control the order in which the road is used.
+Traffic light systems play a crucial role in controlling the flow of vehicles and pedestrians at intersections.
+A Programmable logical controller (PLC) is employed in place of human to control this flow by managing the timing and sequencing of traffic lights.
```
Comment (LateefAkinola) [inline README.md:17]: We are not using a PLC, I think. We are just using Arduino, which is a microcontroller, to mimic what PLC could have executed.

File: README.md [hunk 2]
```diff
@@ -1 +1,30 @@
-# 4-way-Traffic-Light-Prototype-with-Arduino
\ No newline at end of file
+# A 4-way-Traffic-Light-Prototype-with-Arduino
+
+This project aims at mimicking the process of controlling **a 4-way Traffic light system** using a Programmable Logical Controller
```
Comment (LateefAkinola) [inline README.md]: a Micro controller rather ( in this case; Arduino Uno)

File: README.md (hunk 1 above)
Comment (LateefAkinola) [inline README.md:17]: We are not using a PLC, I think. We are just using Arduino, which is a microcontroller, to mimic what PLC could have executed.

Comment (Joshezzy) [reply README.md:17]: That is what I stated, to mimic how a PLC works. MAybe it wasn't clear enough. I will improve on that statement

**Review by LateefAkinola — CHANGES_REQUESTED** [review]
**This is cool. Thank you, Josh.**

Subsequently, kindly add other subheadings like: 

> - 'List of Materials' as seen in @Reedwarn 's commit.
> - 'Additional Materials' (We'd specify other materials not in the circuit, like toy cars, board, etc)

File: README.md (hunk 1 above)
Comment (LateefAkinola) [reply README.md:17]: Would like that. Thank you

**Review by LateefAkinola — APPROVED** [review]
better. Thanks

**Pull request merged**